src/pybind/mgr/dashboard/frontend/src/app/shared/enum/dashboard-promqls.enum.ts

```diff
@@ -8,3 +8,10 @@ export enum Promqls {
   WRITECLIENTTHROUGHPUT = 'sum(rate(ceph_pool_wr_bytes[$interval]))',
   RECOVERYBYTES = 'sum(rate(ceph_osd_recovery_bytes[$interval]))'
 }
+
+export enum RgwPromqls {
+  RGW_REQUEST_PER_SECOND = 'sum(rate(ceph_rgw_req[$interval]))',
```

Comment:
Lets use $interval as 1m, which is static for all queries

Reply:
Done